react-frontend/src/css/casetemplates.css

```diff
@@ -0,0 +1,33 @@
+.pageContainer {
+    padding-top: 100px;
+    background-color: #F2F2F2;
+    padding-bottom: 70px;
+}   
+
+.pageBody {
+    width: 85%;
+    margin: auto;
+}
+
+.pageSubtitle {
+    font-size: 22pt;
+    font-weight: bold;
+    margin-bottom: 24px;
+}
+
+@media only screen and (max-width: 800px) {
+
+    .pageContainer {
+        padding-top: 65px;
+    }
+    .pageBody {
+        width: 330px;
+    }
+
+    .cardAdjustment {
+        margin-left: -8px;
```

Comment:
this hurts me but im okay with it now. Negative margins are not used too often for positioning